src/ColourSchemeChecker.scss

```
@@ -0,0 +1,37 @@
+// Possible colour schemes - see the colourSchemeChecker.txt 
+//Currently using 1. 
```

Space at beginning of comment
```suggestion
// Currently using 1. 
```